tsdb/head.go

```diff
@@ -169,6 +170,10 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 			Name: "prometheus_tsdb_wal_corruptions_total",
 			Help: "Total number of WAL corruptions.",
 		}),
+		walTotalReplayDuration: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "prometheus_tsdb_wal_total_replay_duration_seconds",
```

Comment:
Because it includes not just WAL, but also m-mapped chunks and will have chunks snapshot in future.
```suggestion
			Name: "prometheus_tsdb_wal_total_replay_duration_seconds",
```

Reply:
Did you forget your suggestion?

Reply:
:facepalm:, I meant to suggest `prometheus_tsdb_data_replay_duration_seconds`